src/components/App.js

```diff
@@ -22,7 +22,7 @@ class App extends React.Component {
     /* eslint-disable */ 
```

Comment:
**[OPTIONAL]**
disabling eslint will lead to you missing the object of that rule. This is raising an error because the operation defined but never used.